_rules/invalid-form-field-value-36b590.md

```diff
@@ -0,0 +1,192 @@
+
+## Expectation 2
+
+For each test target with a [form field error indicator][], at least one of the [form field error indicators][form field error indicator] describes the cause of the error, or how to resolve it, in [text][] that is [visible][].
+
+## Expectation 3
+
+For each test target with a [form field error indicator][], at least one of the [form field error indicators][form field error indicator] describes the cause of the error, or how to resolve it, in [text][] that is [included in the accessibility tree][] or included in the [accessible name][] or [accessible description][] of the test target.
+
+## Assumptions
+
+This rule assumes that the apparent cause of a [form field error indicator][] is the actual cause for the [form field error indicator][] being shown. If the actual cause is different, then this rule may generate incorrect results.
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [Understanding Success Criterion 3.3.1: Error Identification](https://www.w3.org/WAI/WCAG21/Understanding/error-identification)
+- [G84: Providing a text description when the user provides information that is not in the list of allowed values](https://www.w3.org/WAI/WCAG21/Techniques/general/G84)
+- [G85: Providing a text description when user input falls outside the required format or values](https://www.w3.org/WAI/WCAG21/Techniques/general/G85)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The `input` element has a [form field error indicator][] that identifies it (by referencing its label) and describes the cause of the error.
+
+```html
+<form>
+	<label for="age">Age (years)</label>
+	<input type="number" id="age" value="0"/>
+	<span id="error">Age must be at least 1</span><br />
+	<input type="button" value="Submit" />
+</form>
+```
+
+#### Passed Example 2
+
+The multiple `input` elements share a [form field error indicator][] that identifies the elements unfilled (by referencing their labels) and describes the cause of the error.
+
+```html
+<form>
+	<h2 id="error">You must fill the name field. You must pick a color.</h2>
+	<label for="name">Name (required)</label>
+	<input type="text" id="name" required />
+	<br />
+	<label for="address">Address</label>
+	<input type="text" id="address" />
+	<p>Pick a color (required)</p>
+	<label><input type="radio" name="color" value="blue" required />Blue</label>
+	<label><input type="radio" name="color" value="yellow" />Yellow</label>
+	<br />
+	<input type="button" value="Submit" />
+</form>
+```
+
+#### Passed Example 3
+
+The `input` element does not have a [form field error indicator][].
+
+```html
+<form>
+	<label for="age">Age (years)</label>
+	<input type="number" id="age" />
+	<input type="button" value="Submit" />
+</form>
+```
+
+### Failed
```

Comment:
You might want to include an example that fails because although it references the label in the error message, there are multiple fields with the same label (but different fieldsets that provide further context), because it doesn't "allow for the identification of" the correct input element.

Reply:
Good suggestion